.github/workflows/build-pipeline-scenarios.yml

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: Elyra Pipeline Scenarios
+on:
+  push: # all branches
+  pull_request: # all branches
+  schedule:
+    # once a day at 4 am (UTC) (8 pm (PST))
+    - cron:  '0 4 * * *'
+
+env:
+  FORCE_COLOR: true
+
+jobs:
+
+  test-producer-consumer:
+    name: Produce / Consumer scenario
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install
+        run: |
+          pip install pandas
+          make install-server
+          make test-dependencies
+      - name: Config Elyra Environment
+        run: |
+          elyra-metadata install runtimes --replace=true \
+                 --schema_name=kfp \
+                 --name=kfp-config \
+                 --display_name="Kubeflow Pipeline (foo)" \
+                 --api_endpoint=http://foo.domain.com/pipeline \
+                 --engine=Argo \
+                 --cos_endpoint=http://foo.domain.com \
+                 --cos_username=user \
+                 --cos_password=pwd123456 \
+                 --cos_bucket=covid
+          elyra-metadata install  runtime-images --replace=true \
+              --schema_name=runtime-image \
+              --name=elyra-demo-anaconda \
+              --display_name="Demo with Anaconda Python 3" \
+              --image_name="elyra/elyra-example-anaconda-py3:latest"
+      - name: Test Producer Consumer
+        run: |
+          WORK_DIR=${GITHUB_WORKSPACE}/build
+          git clone https://github.com/lresende/elyra-sample.git $WORK_DIR/elyra-sample
+          cd $WORK_DIR/elyra-sample/producer-consumer
+          elyra-pipeline run pipelines/producer-consumer.pipeline
+          # ${GITHUB_WORKSPACE}/etc/scripts/run-pipeline-scenarios.sh
```

Comment:
Do we intend to run the script?  If not, we should also add a statement to perform validation as well (like the script does).

Reply:
that is one of the open questions I have, but I might use the script as a way to create a repeatable `function` in Github without having to duplicate a lot of code

Reply:
See [comment](https://github.com/elyra-ai/elyra/pull/2112#issuecomment-915467933)